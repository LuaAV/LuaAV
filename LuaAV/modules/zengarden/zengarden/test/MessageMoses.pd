#N canvas 611 170 450 300 10;
#X obj 21 19 loadbang;
#X obj 21 184 print left;
#X obj 99 184 print right;
#X obj 82 18 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 21 65 t b b b b b;
#X obj 21 147 moses 10;
#X msg 59 103 40;
#X msg 21 103 30;
#X msg 137 103 -10;
#X msg 98 103 -1;
#X connect 0 0 4 0;
#X connect 3 0 4 0;
#X connect 4 0 8 0;
#X connect 4 1 9 0;
#X connect 4 2 7 0;
#X connect 4 3 6 0;
#X connect 4 4 7 0;
#X connect 5 0 1 0;
#X connect 5 1 2 0;
#X connect 6 0 5 1;
#X connect 7 0 5 0;
#X connect 8 0 5 0;
#X connect 9 0 5 1;
