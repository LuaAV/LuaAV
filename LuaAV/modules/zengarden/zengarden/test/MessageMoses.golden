[@ 0.000ms] right: 30
[@ 0.000ms] left: 30
[@ 0.000ms] left: -10